Templates/Kotlin/ResponseType.stencil

```diff
@@ -29,15 +29,13 @@ data class {{ name }}(
         {% endfor %}
     )
 
+    {% if fragmentSelections %}
     companion object {
-        const val typeName = "{{ graphQLName }}"
```

Comment:
Is it useful to keep this from an API POV? 🤔 Probably not if it means keeping companion objects around on every response.